chrony.yaml

```diff
@@ -0,0 +1,78 @@
+package:
+  name: chrony
+  version: 4.3
+  epoch: 0
+  description: NTP client and server programs
+  copyright:
+    - license: GPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - wolfi-baselayout
+      - ca-certificates-bundle
+      - build-base
+      - asciidoctor
+      - libcap-dev
+      - texinfo
+      - nettle-dev
+      - gnutls-dev
+      - libseccomp-dev
+      - bash
+      - busybox
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha512: 1394bac3ed684352fe89b7fef7da50e61f9f522abee807627ae1fc4c2dde891017bc8e5b13759fced028f3a1e875d5e4e5a4f85de65c63b5f83d0ca03bb4c5df
+      uri: "https://download.tuxfamily.org/chrony/chrony-${{package.version}}.tar.gz"
+
+  - runs: |
+      ./chrony/chrony.pre-install
```

Comment:
Why?  Define the `chrony` user as part of the build environment definition instead.